fedgov-cv-message-builder/src/main/java/gov/usdot/cv/msg/builder/IntersectionSituationDataBuilder.java

```diff
@@ -581,6 +591,31 @@ private NodeListXY buildNodeList(IntersectionInputData isdInputData, DrivingLane
 					}
 				}
 
+				if (laneNode.speedLimitType != null && laneNode.speedLimitType.length > 0) {
```

Comment:
For code segments like this inline documentation can help new developers or developers returning to this logic understand the logic and intention better